/**:
  ros__parameters:
    # Parameters that can be modified during runtime
    # depth_module.profile: "640,480,5"
    depth_module.profile: "480,270,5"
    depth_module.enable_auto_exposure: true
    # rgb_camera.profile: "640,480,5"
    rgb_camera.profile: "480,270,5"
    rgb_camera.enable_auto_exposure: true
    enable_depth: true
    enable_color: true
    enable_infra1: false
    enable_infra2: false
    enable_fisheye1: false
    enable_fisheye2: false
    enable_gyro: true
    enable_accel: true
    enable_pose: false
    enable_sync: false
    tf_publish_rate: 0.0

    # Parameters that cannot be changed in runtime:
    serial_no: ""
    usb_port_id: ""
    device_type': ""
    reconnect_timeout: 6.0
    wait_for_device_timeout: -1.0
    rosbag_filename: ""
    initial_reset: false
    unite_imu_method: 1
    clip_distance: -1.0
    publish_tf: true
    diagnostics_period: 0.0

    # Post-Processing Filters 
    align_depth.enable: false
    colorizer.enable: false
    pointcloud.enable: true
    # pointcloud.enable: false
    pointcloud.stream_filter: 2
    pointcloud.allow_no_texture_points: false
    pointcloud.ordered_pc: false
    pointcloud.stream_index_filter: 0
    hdr_merge.enable: false
    depth_module.exposure.1: 7500
    depth_module.gain.1: 16
    depth_module.exposure.2: 1
    depth_module.gain.2: 16

    # Other
    camera_name: "rs_d435i"
    json_file_path: ""
    gyro_fps: 30
    accel_fps: 30
    pose_fps: 200